apps/AEPSampleApp/extensions/TargetView.tsx

```diff
@@ -0,0 +1,200 @@
+      targetProduct,
+      targetOrder,
+    );
+    Target.retrieveLocationContent(locationRequests, parameters);
+  };
+
+  const locationsDisplayed = () =>
+    Target.locationsDisplayed(['clickTestRyan', 'clickTestRyan'], null);
+
+  const locationClickedWithName = () => {
+    const purchaseIDs = ['34', '125'];
+
+    const targetOrder = new TargetOrder('ADCKKIM', 344.3, purchaseIDs);
+    const targetProduct = new TargetProduct('24D3412', 'Books');
+    const profileParameters1 = {ageGroup: '20-32'};
+    const parameters = new TargetParameters(
+      {parameters: 'parametervalue'},
+      profileParameters1,
+      targetProduct,
+      targetOrder,
+    );
+
+    Target.locationClickedWithName('clickTestRyan', parameters);
+  };
+
+  const prefetchContent = () => {
+    const mboxParameters1 = {status: 'platinum'};
+    const mboxParameters2 = {userType: 'Paid'};
+    const purchaseIDs = ['34', '125'];
+
+    const targetOrder = new TargetOrder('ADCKKIM', 344.3, purchaseIDs);
+    const targetProduct = new TargetProduct('24D3412', 'Books');
+    const parameters1 = new TargetParameters(mboxParameters1, null, null, null);
+    const prefetch1 = new TargetPrefetchObject('clickTestRyan', parameters1);
+
+    const parameters2 = new TargetParameters(
+      mboxParameters2,
+      {profileParameters: 'parameterValue'},
+      targetProduct,
+      targetOrder,
+    );
+    const prefetch2 = new TargetPrefetchObject('mboxName2', parameters2);
+
+    const prefetchList = [prefetch1, prefetch2];
+    const profileParameters1 = {ageGroup: '20-32'};
+
+    const parameters = new TargetParameters(
+      {parameters: 'parametervalue'},
+      profileParameters1,
+      targetProduct,
+      targetOrder,
+    );
+    Target.prefetchContent(prefetchList, parameters)
+      .then(success => console.log(success))
+      .catch(err => console.log(err));
+  };
+
+  return (
+    <View style={styles.container}>
+      <ScrollView contentContainerStyle={{marginTop: 75}}>
+        <Button onPress={() => navigation.goBack()} title="Go to main page" />
+        <Text style={styles.welcome}>Target Test App</Text>
+        <Button title="extensionVersion()" onPress={targetExtensionVersion} />
+        <Button
+          title="clearPrefetchCache()"
+          onPress={() => clearPrefetchCache()}
+        />
+        <Button title="getSessionId()" onPress={getSessionId} />
+        <Button title="getThirdPartyId()" onPress={getThirdPartyId} />
+        <Button title="getTntId()" onPress={getTntId} />
+        <Button title="resetExperience()" onPress={resetExperience} />
+        <Button
+          title="setPreviewRestartDeeplink(...)"
+          onPress={setPreviewRestartDeeplink}
+        />
+        <Button title="setSessionId(...)" onPress={setSessionId} />
+        <Button title="setThirdPartyId(...)" onPress={setThirdPartyId} />
+        <Button title="setTntId(...)" onPress={setTntId} />
+        <Button
+          title="retrieveLocationContent(...)"
```

Comment:
Can you double check "retrieveLocationContent(...), prefetchContent(...)  and locationClickedWithName(...)"  I got exception when click these buttons.